server/channels/api4/ldap_test.go

```diff
@@ -299,7 +299,7 @@ func TestAddUserToGroupSyncables(t *testing.T) {
 	id := model.NewId()
 	user := &model.User{
 		Email:       "test@localhost",
-		Username:    model.NewUsername(),
+		Username:    model.NewId(),
```

Comment:
1/5: I think we might want to not revert the various changes replacing `model.NewId` with `model.NewUsername` to reduce the scope of the reverted changes and in case we ever try to revisit this in the future. The only part of this that's causing problems is the change to `model.IsValidUsername`, so I think it should be enough to just revert it and its tests